docs/sources/user/MacOS-Source-Release.md

```diff
@@ -44,6 +44,7 @@ source ~/plaso_env/bin/activate
 Install the plaso dependencies:
 ```
 cd /tmp/plaso-20190708/
+curl -Lo requirements.txt https://raw.githubusercontent.com/log2timeline/plaso/master/requirements.txt
```

Comment:
This pull in requirements.txt from HEAD you likely want to pull in from the tag (20190708)

https://github.com/log2timeline/plaso/blob/20190708/requirements.txt

Reply:
Also doesn't the "Source code (.tar.gz)" package contain the requirements.txt ?

Reply:
You could do that to keep the example consistent. The only reason I didn't do that is because at the top of the page it references pulling the recent release from `https://github.com/log2timeline/plaso/releases`. If you used the tag version (20190708) you could miss some dependencies which are required in the current release.

Reply:
@joachimmetz, unfortunately, the current release ([plaso-20200430](https://github.com/log2timeline/plaso/releases/tag/20200430)) doesn't.

![image](https://user-images.githubusercontent.com/19796681/84116686-a2f2bf00-aa73-11ea-955b-01b4ada5e293.png)

Reply:
Correct "release" as in git tag not as in git HEAD. The problem with using HEAD is that it is a moving target, dependencies change. A requirements.txt of HEAD might not work for a previous git tag.

Also the "Source code (.tar.gz)" package contains the requirements.txt file, what is the reason for downloading the file separately?

Reply:
Ah OK. The [source package](https://github.com/log2timeline/plaso/releases/download/20200430/plaso-20200430.tar.gz) doesn't include the `requirements.txt` file.

You are correct, the page does reference the "Source code (.tar.gz)" package:
> Attached to the most recent release (as of this time version 20190708) is a “**Source code (tar.gz)**” file.

Reply:
Yeah, unfortunately that is part of github and we cannot really fix that UX. But maybe adding the link to the documentation would/could help?

Reply:
Yeah, I think it would help getting that in as a hyperlink at the top. Let me have a go at cleaning up the top of the doc and removing the curl command. I'll update this Pull request.